client/src/main/java/se/fortnox/reactivewizard/client/HttpClientConfig.java

```diff
@@ -163,4 +165,12 @@ public int getReadTimeoutMs() {
     public void setReadTimeoutMs(int readTimeoutMs) {
         this.readTimeoutMs = readTimeoutMs;
     }
+
+    public boolean isUnsafeSecure() {
```

Comment:
There's  a test for HttpClientConfig in HttpClientConfigTest, make sure to get coverage for this there as well :)